### PR TITLE
Update webinterface-wifi to v2.0.0

### PR DESCRIPTION
**Pull request by rM-self-serve**
[Release Notes v2.0.0](https://github.com/rM-self-serve/webinterface-wifi/releases/tag/v2.0.0)
[Docs](https://github.com/rM-self-serve/webinterface-wifi)

Quite a bit of new functionality. 

Should work on all devices+versions.

For anyone testing this, there are [test ssl certs in the repository](https://github.com/toltec-dev/toltec/pull/github.com/rM-self-serve/webinterface-wifi/tree/master/ssl_test_cert). It is necessary the wifi is connected and the web interface is accessible at 10.11.99.1. The following tests should suffice.

---
```
$ webint-wifi net-info
# ensure the wifi ssid is correct 
```
---
```
$ systemctl start webinterface-wifi
```
```
$ webint-wifi create-login
# enter a username/password 
```
```
$ webint-wifi edit
# change [undefined_networks].login_enforced to true
```
```
$ webint-wifi reload
```
Then navigate to the wifi ip address in a browser, enter the credentials, and download a file.

### CODE REVIEW

**Review by Eeems — CHANGES_REQUESTED** [review]
Upon testing, trying to enable the service doesn't work as it's trying to run `/opt/bin/webinterface-wifi --run 80` which is no longer valid.

It may also be worth adding some more helpful output to `configure()` to point the user in the right direction for configuring it?

**Comment by rM-self-serve** [comment]
Sorry for that oversight, the correct service file should now be installed.  

I mentioned where to find usage and docs locally and remotely.

Thanks for testing! I was not aware of the release window so my apologies for cutting it so close.

**Comment by Eeems** [comment]
> Sorry for that oversight, the correct service file should now be installed.
> 
> I mentioned where to find usage and docs locally and remotely.

No worries, this is why we test :stuck_out_tongue: 

> 
> Thanks for testing! I was not aware of the release window so my apologies for cutting it so close.

No worries about the release window. I just picked the earlier one as I don't have to rely on another maintainer to test this.

File: package/webinterface-wifi/package [hunk 1]
```diff
@@ -2,43 +2,65 @@
...
 configure() {
+    [[ -f "$_configdir/config.toml" ]] \
+        || cp "$_etcdir/docs/config.default.toml" \
+            "$_configdir/config.toml"
+
     systemctl daemon-reload
 
     echo ""
-    echo "Run the following command to use $pkgname"
-    how-to-enable "$pkgname.service"
```
Comment (Eeems) [inline package/webinterface-wifi/package:41]: I think it's still worth leaving this in as it's standard output for toltec.

**Comment by rM-self-serve** [comment]
Sounds good to me!

**Comment by Eeems** [comment]
Okay, testing this, and I realized it doesn't automatically restart the service if it's already enabled and running. Since this isn't a launcher which could have various applications running in an unsaved state, it should be safe to restart the service.

```bash
if is-active webinterface-wifi.service; then
    systemctl restart webinterface-wifi
fi
```

**Review by Eeems — APPROVED** [review]

**Pull request merged**